### DISCUSSION TITLE
Vulnerability issue in dropwizard.testing 2.1.1

### DISCUSSION THREAD

**Discussion by zZToasterZz** [dropwizard/dropwizard#5868] [General]
After running the OWASP Dependency checker against the dropwizard, one vulnerability is raised against dropwizard-testing dependency which is brought in:
jakarta.annotation-api-1.3.5.jar CVE-2022-31569

<img width="469" alt="image" src="https://user-images.githubusercontent.com/79910936/190059108-72304fb9-14fc-4209-bfbb-ae755d2d4ab7.png">

**Comment by joschi** [comment]
Dropwizard 2.1.2 has been released recently: https://github.com/dropwizard/dropwizard/releases/tag/v2.1.2

Also, [CVE-2022-31569](https://nvd.nist.gov/vuln/detail/CVE-2022-31569) has been rejected:

> CVE has been marked "REJECT" in the CVE List. These CVEs are stored in the NVD, but do not show up in search results.
